test/benchmark.js

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env node
+// -*- js -*-
+
+"use strict";
+
+var createHash = require("crypto").createHash;
+var fork = require("child_process").fork;
+var get = require("https").get;
+var args = process.argv.slice(2);
+if (!args.length) {
+    args.push("-mc", "warnings=false,unsafe,sequences=1000000");
+}
+args.push("--stats");
+var urls = [
+    "https://code.jquery.com/jquery-3.1.1.js",
+    "https://code.angularjs.org/1.6.1/angular.js",
+    "https://cdnjs.cloudflare.com/ajax/libs/mathjs/3.9.0/math.js",
+    "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.js",
```

Comment:
If you can find me download URLs to those - my Google-fu failed me last night when I tried... 😞